src/DependencyInjection/Xsd2PhpExtension.php

```diff
@@ -50,6 +50,11 @@ public function load(array $configs, ContainerBuilder $container)
             }
         }
 
+        if ($config['configs_jms']) {
```

Comment:
```suggestion
        if (isset($config['configs_jms'])) {
```

Reply:
otherwise it will never get to have `false`

Reply:
bad

Reply:
configs_jms is an array, always with default